code/src/App.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React, { useState } from "react";
+
+import Survey from "./components/survey/Survey";
+import Summary from "./components/Summary";
 
 export const App = () => {
+  const [showSummary, setSummary] = useState(false);
+  const [data, setData] = useState({});
+
+  const handleSubmit = (data) => {
+    setData(data);
+    setSummary(true);
+  };
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+      {showSummary ? <Summary {...data} /> : <Survey onSubmit={handleSubmit} />}
+    </>
```

Comment:
You don't need to wrap it with React Fragment since you return only one component

Reply:
solved